apps/codeforafrica/src/pages/api/admin/config.json

```diff
@@ -0,0 +1,52 @@
+{
+  "backend": {
+    "name": "git-gateway",
+    "branch": "main"
+  },
+  "media_folder": "public/images",
+  "public_folder": "/images",
+  "local_backend": true,
+  "collections": [
+    {
+      "name": "pages",
+      "label": "Pages",
+      "files": [
+        {
+          "label": "Index",
+          "name": "index",
+          "widget": "object",
+          "file": "content/pages/index.md",
+          "fields": [
+            {
+              "label": "Hero",
+              "name": "hero",
+              "widget": "object",
+              "fields": [
+                {
+                  "label": "Title",
+                  "name": "title",
+                  "widget": "string"
```

Comment:
Since we need this as html, I think lets just use `markdown` widget (or `code` if you're feeling brave).

Reply:
I'll go with markdown for now 😀